server/commands/accountProvisioner.js

```diff
@@ -53,13 +53,15 @@ export default async function accountProvisioner({
 }: Props): Promise<AccountProvisionerResult> {
   let result;
   try {
-    result = await teamCreator({
-      name: teamParams.name,
-      domain: teamParams.domain,
-      subdomain: teamParams.subdomain,
-      avatarUrl: teamParams.avatarUrl,
-      authenticationProvider: authenticationProviderParams,
-    });
+    result =
+      (await findExistingTeam(authenticationProviderParams)) ||
```

Comment:
Move this to the top of `teamCreator` rather than exporting and using here and I think we're good – are you able to add some basic tests around these expectations?

Reply:
sure can do!

Reply:
@indietyp How is the progress on these changes?

Reply:
Sorry, I completely forgot to implement these changes. Our company moved on from evaluating Outline so this was subsequently shifted out of my focus and into my freetime. I am hoping to be able to finish the review next Tuesday.